source/MaterialXContrib/MaterialXNode/MaterialXShadingNodeImpl.cpp

```diff
@@ -195,13 +196,14 @@ void bindEnvironmentLighting(MHWRender::MShaderInstance& shader,
     // Environment matrix
     if (parameterList.indexOf(mx::HW::ENV_MATRIX.c_str()) >= 0)
     {
-        const float yRotationPI[4][4]
+        static const float yRotationPI[4][4]
```

Comment:
Nitpick: static naming convention would be Y_ROTATION_PI.

Reply:
Good point!